styleguide/derek/_partials/ticker.ejs

```diff
@@ -0,0 +1,12 @@
+<section class="ticker">
+  <section class="container">
+    <div class="row">
```

Comment:
Oh your `make-container()` mixin hasn't been merged in yet.  So I'd say we either just stick with `container` and add `ticker-row` or add that mixin to this PR instead of the basement PR and then just rebase that one later (as it needs a rebase anyway).